docs/advanced/input_files/input-main.md

```diff
@@ -3275,6 +3277,22 @@ These variables are used to control berry phase and wannier90 interface paramete
   - False: do not output dipole.
 - **Default**: False
 
+### out_current
+
+- **Type**: Boolean
+- **Description**:output current in tddft velocity gague calculation
```

Comment:
change 'tddft' to 'real time TDDFT'

Reply:
change 'in tddft velocity gauge calculation' to ' in real time TDDFT simulations with the velocity gauge'